modules/installation-osp-download-image-restricted.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
```

Comment:
It doesn't look like this module is included in this assembly, should it be? I wasn't sure if the above module was the newer one (and this one should be removed) since the procedures seem to end the same way.